OpenRA.Test/OpenRA.Test.csproj

```diff
@@ -14,7 +14,7 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x86'">
     <DebugSymbols>true</DebugSymbols>
-    <OutputPath>..\</OutputPath>
+    <OutputPath>bin\Debug\</OutputPath>
```

Comment:
I get very suspicious when I see prs fiddle with the csproj configuration.  We have a long history of well-intended changes completely breaking things for other people who are running a different IDE.